### DISCUSSION TITLE
Getting this prompt after verification is approved: ""You were last verified as a on Dec 17, 2024. ""

### DISCUSSION THREAD

**Discussion by nfal1** [community/community#147378] [GitHub Education] [answered]
### Hi everyone,

So I applied for a Student developer pack, and while my affililiation is with University of Crete (UoC), it mistakenly takes my email as if it is from Technical University of Crete (TUC). Nevertheless, my application got approved, however I  get "" You were last verified as a on Dec 17, 2024. "" when checking the page of the application, which after further reading I was guided to upload it as  a discussion here. I am uploading screen snips of both the message and the application. Thanks in advance guys.
![stisup2](https://github.com/user-attachments/assets/f8077743-2530-4893-a7b3-f41fe10c2aaf)
![gitsupport](https://github.com/user-attachments/assets/1f90fd28-a465-4602-b578-d0735be4e678)

**Comment by davevad93** [comment, marked as answer]
Hi @nfal1 ,

From what I see in your screenshots, you don't have to worry because your application has been approved, when the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @nfal1 was approved and your benefits are now available.`

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640, #145995

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
